pymine/logic/parsers/minecraft/vec3.py

```diff
@@ -0,0 +1,22 @@
+from pymine.api.errors import ParsingError
+from pymine.api.abc import AbstractParser
+from pymine.types.player import Player
+
+class Vec2(AbstractParser):
+    def __init__(self) -> None:
+        pass
+
+    def parse(self, s: str, p:[Player, None]) -> tuple:
+        a = s.split()[0]
```

Comment:
Again, you should probably store the result of `.split()` so you don't waste system resources